src/api/process-manager.ts

```diff
@@ -131,7 +133,13 @@ export function startBuild(data: any): Promise<any> {
               data.build_id = build.id;
               delete data.repositories_id;
               delete data.pr;
-              insertBuildRun(data);
+              insertBuildRun(data)
+                .then(() => {
+                  let gitUrl =
+                    `https://api.github.com/repos/${data.head_full_name}/statuses/${data.sha}`;
```

Comment:
can we set this GitHub URL to not be hardcoded like it is now. If you are using GitHub Enterprise on your own servers the URL is different. I think that should not be hardcoded. 

I will investigate on that and make a PR after this one is merged.